lib/zendesk_apps_support/package.rb

```diff
@@ -29,19 +30,15 @@ def validate(marketplace: true)
         errors << Validations::Marketplace.call(self) if marketplace
 
         errors << Validations::Manifest.call(self)
-
         if has_manifest?
           errors << Validations::Source.call(self)
           errors << Validations::Translations.call(self)
+          errors << Validations::Requirements.call(self)
 
-          unless manifest.requirements_only?
+          if !manifest.requirements_only? && !manifest.marketing_only? && !manifest.iframe_only?
```

Comment:
i thought it was a style thing, but reading up on it now it doesn't say anything about this sort of usage.